src/server/db/__tests__/UserPrivate.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @jest-environment node
+ */
+import UserDBPrivate from '../mongo/user-privat-provider'
+
+const storage = UserDBPrivate
+
+const testUser = { identifier: '01', fullName: 'mongo_test', email: 'test@test.test', mobile: '123456789' }
+jest.setTimeout(30000)
+describe('UserPrivate', () => {
+  afterAll(async () => {
+    await storage.model.deleteMany({ fullName: new RegExp('mongo_test', 'i') })
+  })
+
+  it('Should addUser user', async () => {
+    let res = await storage.addUser(testUser)
+    expect(res).toBeTruthy()
+  })
+
+  it('Should updateUser user', async () => {
+    let res = await storage.updateUser({ identifier: testUser.identifier, jwt: 'test jwt' })
+    expect(res).toBeTruthy()
+
+    let user = await storage.getByIdentifier(testUser.identifier)
+    expect(user).toBeTruthy()
+
+    expect(user.jwt === 'test jwt').toBeTruthy()
```

Comment:
you need to check that other fields didnt change

Reply:
done